engines/csvengine.py

```diff
@@ -53,27 +55,23 @@ def create_db(self):
     def create_table(self):
         """Create the table by creating an empty csv file"""
         self.auto_column_number = 1
-        self.output_file = open(self.table_name(), "w")
-        self.output_file.write(
-            ','.join(['"%s"' % c for c in self.table.get_insert_columns(join=False,create=True)]))
+        self.file = open(self.table_name(),'w')
+        self.output_file = csv.writer(self.file, dialect='excel', lineterminator='\n', quoting=csv.QUOTE_MINIMAL)
+        self.output_file.writerow(self.table.get_insert_columns(join=False,create=True))
+        self.table_names.append((self.file, self.table_name()))
```

Comment:
Does the 2nd item in the tuple get used anywhere? If not, what about just a list of file names called `file_names`?